pandas/core/groupby.py

```diff
@@ -598,7 +610,7 @@ def get_group(self, name, obj=None):
         if obj is None:
             obj = self._selected_obj
 
-        inds = self._get_index(name)
+        inds = self._get_index(name, raise_on_missing=True)
```

Comment:
I was a little worried that there could be a key in self.indices which has an empty array as its value (like a categorial grouper with not all categories represented), but it looks like that's impossible.